internal/controller/bindata/daemon/04.daemon_cluster_role.yaml

```diff
@@ -19,6 +19,7 @@ rules:
   - get
   - list
   - watch
+  - update
```

Comment:
We should wait until the round-trip PR goes in because there's a substantial change on how the rbac is done there.

Reply:
It should be fine to merge in since it is appending to the end of the list.

Reply:
:sweat_smile: Someone needs to do the rebase, it's either going to be done in this PR which touches rbac, or yours which touches rbac.